src/assets/translations/en-us.yaml

```diff
@@ -4596,6 +4596,13 @@ troubleshooting:
       title: Factory Reset
       description: Factory Reset will remove all Rancher Desktop Configurations.
       buttonText: Factory Reset
+      messageBox:
+        title: Rancher Desktop - Factory Reset
+        message: Perform a factory reset?
+        detail: <p>Doing a factory reset will remove your cluster and all Rancher Desktop settings, and shut down Rancher Desktop. If you intend to continue using Rancher Desktop, you will need to manually start it and go through the initial set up again.</p><p>Are you sure you want to factory reset?</p>
```

Comment:
Verifying the HTML markup is rendered correctly. But I would reword "Are you sure you want to factory reset?" as "Are you sure you want to reset everything?"